redbot/core/cli.py

```diff
@@ -42,9 +42,8 @@ async def interactive_config(red, token_set, prefix_set, *, print_header=True):
     if not token_set:
         print(
             "Please enter a valid token.\n"
-            "You can find out how to obtain a token with this guide"
-            ' (section "Creating a Bot Account"):\n'
-            f"https://discordpy.readthedocs.io/en/v{discord_version}/discord.html#creating-a-bot-account"
+            "You can find out how to obtain a token with this guide:\n"
+            "https://docs.discord.red/en/stable/bot_application_guide#creating-a-bot-account"
```

Comment:
I haven't looked too closely at this link at first, but looks like you missed `.html` extension here:
```suggestion
            "https://docs.discord.red/en/stable/bot_application_guide.html#creating-a-bot-account"
```